ldap/servers/plugins/replication/repl5_replica.c

```diff
@@ -22,6 +22,7 @@
 #include "slap.h"
 
 #define RUV_SAVE_INTERVAL (30 * 1000) /* 30 seconds */
+#define REPLICA_UPDATE_INTERVAL (60 * 1000) /* 1 minute */
```

Comment:
This value seems too frequent to me. In a 4 suppliers topology, the keep alive traffic will be +200 updates per hour that is high compare to regular traffic.

I think a low value is useful for test reason. An option would be to have a replica configuration parameter with a higher default value for example an update per hour. Then for the test we can set it to an update per minute and verify that it occurs.

The rest of the patch looks good to me

Reply:
Active Directory defaults to once every 5 minutes or 20 times per hour, which I think would be enough.

Reply:
Sorry I wanted to mention that about the interval.  I had it set to 1 minute for easier testing.  

The only issue we have is the CI tests. If we bump it to 10, 30, or 60 minutes its going to add a lot of time to the CI test suite.  Or, I adjust the test suite and not check for the next interval.  Our CI tests already take a very long time to run so I am just being cautious.

Reply:
If you add a new replica configuration parameter with a 3600s default value, you may use it to select 60s for testing.

Reply:
@tbordaz true we could do that, but is there value in a new config setting for customers?  There could be I suppose.  

We definitely need to test this feature in our CI, and it would not be a dangerous config setting if misused by an admin (if we set a minimum requirement on the interval).  So I will work on that...